staff/lab/mod-printer

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+"""Remove or add printers to/from CUPS classes."""
+import argparse
+import re
+import subprocess
+import sys
+
+
+def modify_printer(args):
+    """Perform internal call to CUPS CLI to modify class(es)."""
+    # Determine which CUPS classes are available
+    try:
+        lpstat_out = subprocess.run(["lpstat", "-p"], check=True, \
+            stdout=subprocess.PIPE, stdin=subprocess.PIPE).strip().split("\n")
+    except subprocess.CalledProcessError as e:
```

Comment:
IMO, it's not necessary to catch the exception. If the command fails, the exception message that gets printed by default will show the exit code, as well as the full argument list.

(same throughout)